### PR TITLE
On scroll changed event listener

### PR DESCRIPTION
**Pull request by kmathew-pdftron**

### CODE REVIEW

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]
please refer to react native on implementation, it's entirely off track here

**Review by JamieDass — APPROVED** [review]
I think iOS looks ok

**Comment by terryyoon-pdftron** [comment]
@sgong-pdftron

File: android/build.gradle [hunk 1]
```diff
@@ -17,7 +17,7 @@ rootProject.allprojects {
         google()
         mavenCentral()
         maven {
-            url "https://pdftron-maven.s3.amazonaws.com/release"
+            url "https://pdftron-maven.s3.amazonaws.com/snapshot"
```
Comment (sgong-pdftron) [inline android/build.gradle:20]: looks like bad merge here should revert

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]

**Review by sgong-pdftron — APPROVED** [review]

**Review by sgong-pdftron — APPROVED** [review]

**Pull request merged**